### ISSUE TITLE
Terminal commands with urls are not escaped properly.

### ISSUE THREAD

**Issue by plisy** [microsoft/codetour#152]
When adding commands to run in terminal with the simplified syntax

```
>> curl -H "Accept: application/json" https://icanhazdadjoke.com/
```

The output is not clickable or runnable

![image](https://user-images.githubusercontent.com/1951942/112073274-fdf03100-8b30-11eb-8687-7f38f733feea.png)

When trying to use the `command:codetour.sendTextToTerminal?[""]` the URL is the only thing that gets parsed/highlighted.

![image](https://user-images.githubusercontent.com/1951942/112073771-f7ae8480-8b31-11eb-8329-b2949eab8e21.png)

VS Code: 1.54.3 (stable)
CodeTour: v0.0.47

**Comment by lostintangent** [comment]
Hey! This is likely due to a very simplistic/naive regex pattern 😄 Let me take a quick look and try to slip this into my next update. Thanks for reporting it!

**Comment by lostintangent** [comment]
This is fixed in `v0.0.48`, which is being published to the marketplace right now.
